operator/go.mod

```diff
@@ -459,3 +467,8 @@ replace (
 
 	pgregory.net/rapid => github.com/chrisseto/rapid v0.0.0-20240815210052-cdeef406c65c
 )
+
+replace (
+	github.com/google/cel-go => github.com/google/cel-go v0.17.8
```

Comment:
If you happen to recall the reasons these are required or even have the build failure messages, it'd be really helpful to note them as comments. Also include these as part of the replace block above please.

Reply:
I added my reasoning